config/config.go

```diff
@@ -34,6 +34,10 @@ type Config struct {
 		DisableCalendarSubscriptions bool   `public:"true" info:"If set, disables all active calendar subscriptions as well as the ability to create new calendar subscriptions."`
 	}
 
+	Services struct {
+		RequiredLabels []string `public:"true" info:"List of label names to require new services to define."`
+	}
+
```

Comment:
Can we move this in general? If not, can we move related things like "Disable Label Creation" here?

Reply:
Offline convo summary for ref:
- Keeping it under services as other things may be added service-specific later
- Things like DisableLabelCreation will be removed/deprecated